src/helpers.js

```diff
@@ -35,9 +35,13 @@ function reduceError (error = {}) {
  */
 function appendQueryParams (url, qs) {
   let result = url
-  if (qs) {
+  if (qs && !(Object.keys(qs).length === 0)) {
+    const filteredQs = Object.entries(qs).filter(([k, v]) => v)
```

Comment:
Does this filter out false values as well?

Reply:
no. and it should not too. false may not be the default value in all cases

Reply:
fixed